std/experimental/logger/asynclogger.d

```diff
@@ -0,0 +1,141 @@
+module std.experimental.logger.asynclogger;
+
+import std.experimental.logger.core;
+import std.concurrency;
+public import std.typecons : Unique, unique;
+
+/** The $(D AsyncLogger) processes logs asynchronously by using another $(D Logger) implementation
```

Comment:
the first sentences is confusing:

the act of using another logger does not make it asynchronously

please rewrite